main/src/cgeo/geocaching/ui/dialog/CoordinatesCalculateDialog.java

```diff
@@ -0,0 +1,1101 @@
+package cgeo.geocaching.ui.dialog;
+
+import android.app.Dialog;
+import android.os.Bundle;
+import android.support.annotation.IdRes;
+import android.support.v4.BuildConfig;
+import android.support.v4.app.DialogFragment;
+import android.support.v7.widget.GridLayout;
+import android.text.Editable;
+import android.text.TextWatcher;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.Window;
+import android.widget.AdapterView;
+import android.widget.ArrayAdapter;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.HorizontalScrollView;
+import android.widget.ImageButton;
+import android.widget.Spinner;
+import android.widget.TextView;
+
+import org.apache.commons.lang3.SerializationUtils;
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+import butterknife.ButterKnife;
+import cgeo.geocaching.R;
+import cgeo.geocaching.activity.AbstractActivity;
+import cgeo.geocaching.location.Geopoint;
+import cgeo.geocaching.location.GeopointFormatter;
+import cgeo.geocaching.models.CalcState;
+import cgeo.geocaching.sensors.Sensors;
+import cgeo.geocaching.settings.Settings;
+import cgeo.geocaching.ui.CalculateButton;
+import cgeo.geocaching.ui.EditButton;
+import cgeo.geocaching.ui.CalculatorVariable;
+
+import static cgeo.geocaching.R.id.PlainFormat;
+import static cgeo.geocaching.R.id.coordTable;
+import static cgeo.geocaching.models.CalcState.ERROR_CHAR;
+
+public class CoordinatesCalculateDialog extends DialogFragment {
+
+    private static final String PLACE_HOLDER = "~"; // Character used to represent a "blanked-out' CoordinateButton.
+
+    static final String SYMBOL_DEG = "°";
+    static final String SYMBOL_MIN = "'";
+    static final String SYMBOL_SEC = "\"";
+    static final String SYMBOL_POINT = ".";
+
+    private Geopoint gp;
+    private CalcState savedState;
+
+    // Through out this implementation:

```

Comment:
Perhaps.  The reason it's there though is that it's right above the two variables it's discussing.

I was thinking people may well come across those variables else ware in the code and wonder what exactly they were meant to represent so placing the comment directly above them would make it easy to find.  I wouldn't even know how to find the "class javadoc" myself but perhaps someone who knows more about what they are doing would think to look there first. I wouldn't know ;)